js/output/sql/sql-output.js

```diff
@@ -96,6 +96,13 @@ window.SQLOutput = Backbone.View.extend({
             errorMessage = i18n._("You have multiple columns named \"%(colName)s\" - " +
                 "column names must be unique.", {colName});
         }
+        const ambColStr = "ambiguous column name:";
+        const ambColError = sqliteError.indexOf(ambColStr) > -1;
+        if (ambColError) {
+            const colName = errorMessage.split(ambColStr)[1].trim();
+            errorMessage = i18n._("Ambiguous column name \"%(colName)s\".",
```

Comment:
Maybe we could help learners a bit more here and something like this:

`When joining two tables which have columns with an identical name, prefix the column name with the table name like this table1.\"%(colName)s\"`

Reply:
Good idea! I suggest a rewording: `This means that two or more of the tables that you are joining contain a column with that name. When you want to use that column in your query, you need to say which table you are talking about. Example: firstTable.\"%(colName)s\"` Too bad that we can't get the actual table names from the query...

Reply:
Nice user-friendly wording. It might be a bit too long for Yoko OhNoes, so here's a shorter version of that if needed:
"Multiple tables that you're joining contain a column with that name. To use that column in your query, specify the table of the column. Example: firstTable.\"%(colName)s\""

Reply:
@aasmundeldhuset okay, SGTM.
Since this is already live, I'll add this change to the current PR #724 if that's okay. I will make it a separate string so that we do not break existing translations.

Reply:
Thank you @aasmundeldhuset  and @pamelafox. In the end, I created a separate PR #717 and added a missing test.